assets/dev/js/frontend/handlers/section/handles-position.js

```diff
@@ -27,15 +27,21 @@ export default class HandlesPosition extends elementorModules.frontend.handlers.
 			return;
 		}
 
+		const insideHandleClass = 'elementor-section--handles-inside',
```

Comment:
Is this class removed when the user disables scroll snap?

Reply:
No, I thought it will be redundant to add extra complicity to the code.
Does it really matter if the handles will be inside the section until the next page load?
What do you think? @kobizz @DorShahar